src/vl-search-filter.js

```diff
@@ -56,6 +56,10 @@ export class VlSearchFilter extends NativeVlElement(HTMLDivElement) {
   get _elementName() {
     return 'vl-search-filter';
   }
+
+  get formData() {
+    return new FormData(this.querySelector('form'));
```

Comment:
Waarom is dit eigenlijk nodig als publieke getter op search filter? Een afnemer moet toch zelf de form in de search filter steken, dus die weet toch waar die zelf op kan selecteren indien die de formData nodig heeft?

Reply:
Het nut zie ik ook niet direct, de rich table haalt toch zelf de values uit de input velden?

Reply:
Ja, dat klopt inderdaad ook. Laten we die functie, en bijgevolg alles, wat ik hier gedaan heb weg... en regelen we dat hogerop? In dit geval in de rich-data-table?

Reply:
ik zou "iets" geencapsuleerd steken in search-filter, of dat nu formdata moet gebruiken of niet laat ik in het midden.

Reply:
ok, als niemand opnieuw bezwaar indient, wordt het formdata.